hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/bucket/OMBucketSetPropertyRequest.java

```diff
@@ -202,6 +202,12 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
       if (defaultReplicationConfig != null) {
         // Resetting the default replication config.
         bucketInfoBuilder.setDefaultReplicationConfig(defaultReplicationConfig);
+      } else {
```

Comment:
Nit: We can change this to `else if` statement.

Reply:
Thanks for the review @aswinshakil . Addressed the comment.